front/public/locales/en/errors.json

```
@@ -126,6 +126,7 @@
     "paced_train": {
       "BatchNotFound": "'{{count}}' paced train(s) could not be found",
       "Database": "Internal error (database)",
+      "ExceptionNotFound": "Paced train '{{exception_key}}' could not be found",
```

Should it rather be that ?
```suggestion
      "ExceptionNotFound": "Exception '{{exception_key}}' could not be found",
```